caluma_interval/__main__.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+This is the CLI for caluma_interval.
+"""
+
+import argparse
+import logging
+import sys
+
+from envparse import env
+
+from caluma_interval import interval
+
+logger = logging.getLogger(__name__)
+logging.basicConfig()
+
+
+def parse_arguments(args):
+    """
+    Parse all given arguments.
+    :param args: list
+    :return: argparse.Namespace
+    """
+    parser = argparse.ArgumentParser(
```

Comment:
I normally use click instead of argparse for new code. Shorter and more readable :)

Reply:
Personally I prefer argparse over click.

For me it provides more flexibility, is more transparent about what's going on and doesn't need me to design my callable commands for it.

That's just my 10¢. If you insist, I will change it of course!

Reply:
We don't have policy, click is just my preference. So sure thing, keep it :)